src/site/stages/build/process-cms-exports/transformers/node-event_listing.js

```diff
@@ -20,6 +20,7 @@ const reverseFields = reverseFieldListing => ({
           fieldFeatured: reverseField.fieldFeatured,
           fieldDate: reverseField.fieldDate,
           fieldDescription: reverseField.fieldDescription,
+          fieldFacilityLocation: reverseField.fieldFacilityLocation,
```

Comment:
The `fieldFacilityLocation` field should be added to `node-event_listing`'s output schema. Specifically [here](https://github.com/department-of-veterans-affairs/vets-website/blob/b4d32f9a60365ddf37095dd99385c40698bead89/src/site/stages/build/process-cms-exports/schemas/output/node-event_listing.js#L10-L16).